<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lucida Sans Unicode</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>351</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CADASTRO</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>172</y>
      <width>207</width>
      <height>213</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nome">
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>E-mail:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="e_mail">
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Senha:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="senha">
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botao_salvar">
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
